### PR TITLE
Split new integrations into separate cards

### PR DESCRIPTION
**Pull request by timmo001**
## Description

Now that there are many integrations for HA, the new integrations panel is becoming cluttered and hard to follow. This splits the new integrations section into separate cards so the user can find the configure button more easily.

## Screenshots

image

## Checklist

- [x] Code is tested and works on my devices

### CODE REVIEW

**Comment by iantrich** [comment]
I was looking at this the other day and wondering if we should break this out by domains that can be expanded to view integrations available for that domain?

**Comment by bramkragten** [comment]
> I was looking at this the other day and wondering if we should break this out by domains that can be expanded to view integrations available for that domain?

What if an integration has multiple domains?

**Comment by iantrich** [comment]
> What if an integration has multiple domains?

show it in both, I guess

**Comment by balloob** [comment]
So I think that this proposal is ugly. 

I think that we should remove it completely and replace it with a FAB that opens a dialog with `<paper-item>`
